### ISSUE TITLE
[AXE-CORE]: Heading levels SHOULD increase by one - see "By mail", "In person", "With the help"

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6359]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Heading levels **should** increase by one. On `https://www.va.gov/disability/how-to-file-claim/`, there are three subheadings under the `h2` "What other ways can I file my disability claim?" that are coded as `h4`. 

### Why it matters

The underlying purpose of headers is to convey the structure of the page. For sighted users, the same purpose is achieved using different sizes of text. Text size, however, is not helpful for users of screen readers, because a screen reader identifies a header only if it is properly marked-up. When heading elements are applied correctly, the page becomes much easier to navigate for screen reader users and sighted users alike.

In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

The purpose of headings is to describe the structure of the webpage, not just highlight important text. They should be brief, clear, unique, and marked with h1 through h6 elements applied in hierarchical order. All of these qualities make headings valuable tools for screen reader users. Similar to the way sighted users can glance at a page and get a sense of its contents, screen reader users can navigate through headings. Well written and properly ordered headings can save screen reader time and frustration.

In addition to making the page more accessible, headings have other benefits since search engines use headings when filtering, ordering, and displaying results. Improving the accessibility of your site can also have the effect of making your page more findable.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: production

## Steps to Recreate

1. Enter `https://www.va.gov/disability/how-to-file-claim/` in browser
2. Start screenreading device listed in Environment
3. Read through the page, note that a heading level is skipped between the `h2` and `h4` described above.
4. Alternately, run an axe or axe-coconut scan in the browser to verify the issue

## Possible Fixes (optional)

- Use an `h3` in the CMS
- If possible/desired, add an `h4` utility class to retain the styling

## WCAG or Vendor Guidance (optional)

* [Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order?application=AxeChrome)

## Screenshots or Trace Logs

![vsa-pw-ama-heading-levels](https://user-images.githubusercontent.com/57469/75694823-23d13c00-5c77-11ea-9cd0-bd1a1fd6b3b7.png)

**Comment by RLHecht** [comment]
These changes have been made in Drupal. Moving to validate and will close when it's live on the site.

**Comment by jenstrickland** [comment]
@RLHecht - Please let me know when the edits are live and I'll validate, comment, and close. I need to record the issue remediation elsewhere.

**Comment by RLHecht** [comment]
@jenstrickland This is now live: www.va.gov/disability/how-to-file-claim/

**Comment by jenstrickland** [comment]
Thanks, @RLHecht & @AkucherGCIO ! Validated with axe and axe-coconut and no longer see heading errors on this page. :clap:
